config/locales/en.yml

```
@@ -530,8 +525,9 @@ en:
       external_users: External Users
       finishing_rate: Finishing Rate
     submit:
-      failure: An error occurred while transmitting your score. Please try again later.
-      too_late: Your submission was saved successfully but was received after the deadline passed.
+      failure: The scoring was performed successfully but an error occurred while transmitting your score. Please try again later.
+      too_late: Your submission was saved successfully but was received after the deadline passed so that only %{score_sent} points were transmitted.
```

Potentially, we can simplify that a little:

```suggestion
      too_late: Your submission was saved successfully but was received after the deadline, so that only %{score_sent} points were transmitted.
```